genesis-builder/l2_move_template.hbs

```diff
@@ -1,9 +1,16 @@
 module {{name}}::{{snake name}} {
-    use aptos_framework::fungible_asset_u256::
-    {{~#if has_fungible_asset}}{FungibleAsset, zero}{{else}}zero{{/if}};
-    use EthToken::eth_token::get_metadata;
+    {{#if has_fungible_asset}}
```

Comment:
Can we use `{{~if}}` here with only `FungibleAsset` and `zero`? This will append it to the previous line.

Reply:
Done in https://github.com/UmiNetwork/op-move/pull/358/commits/157385c50eb8dc51c7e2486648193597b9f01352